README.md

```diff
@@ -16,14 +16,14 @@
 ## Key features
 
 ArkScript is
-* small: the compiler, and the virtual machines fit under 5000 lines, but also small in term of keywords (it has only 10)!
+* small: the compiler and the virtual machines fit under 5000 lines, but also small in terms of keywords (it has only 10)!
```

Comment:
Just a small comment but you couldn't know (and I'm taking the opportunity to hit two birds in one stone): it's 8000 lines now, not 5000